lib/elixir_search_extractor/file_upload/file_uploads.ex

```diff
@@ -15,10 +16,10 @@ defmodule ElixirSearchExtractor.FileUpload.FileUploads do
     csv_file = attributes["csv_file"]
 
     with :ok <- CsvValidator.validate_file(csv_file),
-         {:ok, upload_path} <- CsvUploader.upload_file_path(user_id, csv_file),
-         {:ok, refactored_attributes} <- refactor_attributes(attributes, user_id, upload_path),
+         {:ok, refactored_attributes} <- refactor_attributes(attributes, user_id, csv_file),
          {:ok, %{create_keyword_file: keyword_file}} <-
-           Repo.transaction(create_and_upload_file_multi(csv_file, refactored_attributes)) do
+           Repo.transaction(create_and_upload_file_multi(csv_file, refactored_attributes)),
+         :ok <- extract_keywords(keyword_file) do
```

Comment:
Maybe we can enqueue job in the multi 🤔  So if the job is failed to insert, it all fail together.

We can still put upload file in the last step because that operation is hard to revert. But for create record/job, DB will be reverted automatically 👍

Reply:
@rosle Why would the job fail to start? What could be the scenario? Also, If a start the job before file upload, and the job doesn't find the file that would pose another problem don't you think? 🤔

Reply:
@rafayet-monon Oban will insert the job record into the db, so it would be possible that it would fail in that step. So I'm thinking of moving the enqueue job into the multi above. 🤔 💭

Reply:
@rosle Thanks for the suggestions. I fixed them in [b8d04d9](https://github.com/rafayet-monon/elixir_search_extractor/pull/39/commits/b8d04d9dd270ca6a54a99354503e14b280b10891)